src/Hl7.Fhir.Specification/Specification/Source/Summary/ArtifactSummaryExtensions.cs

```diff
@@ -6,8 +6,6 @@
  * available at https://github.com/ewoutkramer/fhir-net-api/blob/master/LICENSE
  */
 
-#if NET_FILESYSTEM
```

Comment:
Je zou verwachten dat er filesystem code in de unit staat, en dat het niet compileert onder NETSTD11, toch?

Reply:
HL7.Fhir.Specification targets only net45 and netstandard2.0. So no netstandard1.1 anymore. That's why this conditional directive could be deleted.